test/OrchardCore.Tests/Data/ContentItemTests.cs

```diff
@@ -117,27 +118,34 @@ public void ContentShouldBeJsonPathQueryable()
             AssertJsonEqual(textPropertyNode, contentItemJson.SelectNode("$..Text"));
         }
 
-        private static ContentItem CreateContentItemWithMyPart(string text = "test")
+        [Fact]
+        public void RemovingPropertyShouldWork()
         {
             var contentItem = new ContentItem();
             contentItem.GetOrCreate<MyPart>();
-            contentItem.Alter<MyPart>(x => x.Text = text);
-            
-            return contentItem;
+            contentItem.Alter<MyPart>(x => x.Text = "test");
+
+            JsonDynamicObject content = contentItem.Content;
```

Comment:
We could use `var`

Reply:
The `contentItem.Content` is a dynamic type , specifying the type explicitly here allows for efficient code prompting and compile-time validation

Reply:
Ah, right